helm-deployment/ldap/values.yaml

```diff
@@ -0,0 +1,72 @@
+adminPassword: "admin"
```

Comment:
the same file already exist -> `tests/deployment/ldap/values.yaml`

Reply:
I'll switch to values from tests/